src/lib/block_producer/block_producer.ml

```diff
@@ -530,7 +530,10 @@ let retry ?(max = 3) ~logger ~error_message f =
   go 0
 
 module Vrf_evaluation_state = struct
-  type status = At of Mina_numbers.Global_slot.t | Start | Completed
+  type status =
+    | At of Mina_numbers.Global_slot_since_genesis.t
```

Comment:
`Global_slot_since_hard_fork.t`. The values it is being compared to are `Global_slot_since_hard_fork.t`s

Reply:
Done.